src/coreclr/binder/bindertracing.cpp

```diff
@@ -176,8 +176,8 @@ namespace BinderTracing
 {
     static thread_local bool t_AssemblyLoadStartInProgress = false;
 
-    AssemblyBindOperation::AssemblyBindOperation(AssemblySpec *assemblySpec, const WCHAR *assemblyPath)
-        : m_bindRequest { assemblySpec, nullptr, assemblyPath }
+    AssemblyBindOperation::AssemblyBindOperation(AssemblySpec *assemblySpec, const SString& assemblyPath)
+        : m_bindRequest { assemblySpec, SString::Empty(), assemblyPath }
```

Comment:
Old gcc has trouble with initializer syntax when a candidate ctor of same type is marked `explicit`. Minimal repro: https://godbolt.org/z/c3heh1568